src/agents/supervisor/prompts.py

```diff
@@ -129,3 +130,55 @@
 - Avoid giving any impression regarding the correctness or incorrectness of the rejected responses.
 - Do not add any more headers or sub-headers to the final response.
 """
+
+
+PLANNER_PROMPT_STRUCTURED_OUTPUT = """
```

Comment:
please update and use the original prompt template

Reply:
Done